packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/PairedResultGraph/Reflow/DynamicData.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PairedResultGraph from '@cerner/terra-graphs/lib/components/PairedResult/PairedResultGraph';
+import Button from 'terra-button/lib/Button';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getPairedResultTimeseriesPanningConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/PairedResult/pairedResultPanning';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/PairedResult/dynamicPanningData';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+const initialState = {
+  initial: 0,
+  factor: 3,
+  dataset: [utils.deepClone(data[0])],
+  graphConfig: utils.deepClone(getPairedResultTimeseriesPanningConfig('#PairedResultDynamicData')),
+};
+
+const PairedResultPanningExample = () => {
+  const reducer = (panState, action) => {
+    const newGraphState = utils.deepClone(panState.graphConfig);
+    let hour;
+    const newDataset = [utils.deepClone(data[1])];
+
+    switch (action.type) {
+      case 'panLeft':
+        hour = panState.initial - panState.factor;
+        break;
+      case 'panRight':
+        hour = panState.initial + panState.factor;
+        break;
+      default:
```

Comment:
Are we doing default cases? I added an ignore rule to the linter.

Reply:
I thought it would be better to go with linter rules that is why I added a dummy default case.

Reply:
If that's the case, then return `panData` so the returns are consistent.

Reply:
Correction: `panState`

Reply:
Added [here](https://github.com/cerner/terra-graphs/pull/156/commits/3ec2674b7f7f154b33ac13031d6305c59f25d341#diff-1d9d2676de20d35da344c0163df88371c4fecad82d5fc629ab4bc22992391645R44).